HyperVExtension/DevSetupAgent.sln

```diff
@@ -0,0 +1,213 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.9.34616.47

```

Comment:
Extensions need to go into the <root>\extensions\ directory

Reply:
I spoke to Kristen about this, that change can in after we had already started this process. So I told her we would move it after the changes go in. Would you say it is mandatory  for this to go in?

Reply:
Sounds good, thank you!